src/parser/jobs/drg/modules/Positionals.js

```diff
@@ -0,0 +1,211 @@
+			this.addTimestampHook(event.timestamp + TRUE_NORTH_CD, this._updateTnCharges)
+		}
+	}
+
+	_procSuccess() {
+		// you did it!
+		if (this._currentCombo) {
+			this._currentCombo.success = true
+		}
+
+		// don't actually close until next gcd happens
+	}
+
+	_onComplete() {
+		const missed = this._rtCombos.filter(combo => !combo.success).length
+
+		this.suggestions.add(
+			new TieredSuggestion({
+				icon: ACTIONS.RAIDEN_THRUST.icon,
+				content: <Trans id="drg.positionals.suggestions.content">Performing the proper positionals for <ActionLink {...ACTIONS.FANG_AND_CLAW} /> and <ActionLink {...ACTIONS.WHEELING_THRUST} /> will do more damage and let you use <ActionLink {...ACTIONS.RAIDEN_THRUST} />. Try to land these positionals, and use <ActionLink {...ACTIONS.TRUE_NORTH} /> if it's available.</Trans>,
+				tiers: {
+					1: SEVERITY.MINOR,
+					10: SEVERITY.MEDIUM,
+				},
+				value: missed,
+				why: <Trans id="drg.positionals.suggestions.why">You missed <Plural value={missed} one="# proc" other="# procs" /> of Raiden Thrust.</Trans>,
+			}),
+		)
+	}
+
+	_checkIcon(success, size = 'large') {
+		return success ? <Icon color="green" name="check" size={size} /> : <Icon color="red" name="x" size={size} />
+	}
+
+	_procTable() {
+		return this._rtCombos.filter(combo => !combo.success).map(combo => {
+			const action = getDataBy(ACTIONS, 'id', combo.next)
+
+			return <Table.Row key={combo.time}>
+				<Table.Cell>{this.createTimelineButton(combo.time)}</Table.Cell>
+				<Table.Cell><ActionLink {...action} /></Table.Cell>
+				<Table.Cell textAlign="center">{this._checkIcon(combo.trueNorthCharges > 0, '')} (<Plural id="drg.positionals.tn-charges" value={combo.trueNorthCharges} one="# charge" other="# charges" />)</Table.Cell>
+			</Table.Row>
+		})
+	}
+
+	output() {
+		const missed = this._rtCombos.filter(combo => !combo.success).length
+
+		return <Fragment>
+			<Message>
+				<Trans id="drg.positionals.analysis.message">Being at the rear when using <ActionLink {...ACTIONS.WHEELING_THRUST} /> or on the flank when using <ActionLink {...ACTIONS.FANG_AND_CLAW} /> will allow you to use <ActionLink {...ACTIONS.RAIDEN_THRUST} /> instead of <ActionLink {...ACTIONS.TRUE_THRUST} />. You should be trying to proc this ability as much as possible, relying on <ActionLink {...ACTIONS.TRUE_NORTH} /> in situations where you cannot reach the proper position. The table below displays missed positionals (if any), and whether or not <ActionLink {...ACTIONS.TRUE_NORTH} /> was available to use.</Trans>
+			</Message>
+			<Message info>
+				<Trans id="drg.positionals.analysis.missed"><Icon name="info" /> You missed <strong>{missed}</strong> of <strong>{this._rtCombos.length}</strong> possible <ActionLink {...ACTIONS.RAIDEN_THRUST} /> procs.
+					{missed > 0 ? (
+						<>
+							Of these missed procs, <strong>{this._rtCombos.filter(combo => !combo.success && combo.trueNorthCharges > 0).length}</strong> could be handled with <ActionLink {...ACTIONS.TRUE_NORTH} />
```

Comment:
You're going to need to do this string another way - it isn't getting picked up properly for translation. I'm not 100% sure, but I *think* turning it into two separate strings that you just list one after another should work. So something like:

```js
<Trans id="drg.positionals.analysis.missed"><Icon name="info" /> You missed <strong>{missed}</strong>...</Trans>
{missed > 0 ? (
    <Trans id="drg.positionals.analysis.truenorth">Of these missed procs...</Trans>
) : ('')}
```

Instead of embedding the second half of the string inside the first.

Reply:
According to DT ([link](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/lingui__react/Select.d.ts)), the plural tag accepts react nodes, so this can be handled in a single translation using something like

```
<Trans id="foo">Outer text<Plural value={missed} _0={<>Inner text<>}/>... </Trans<
```
Untested (on phone), but might be worth a shot.

Reply:
Opted for the split `Trans` tags, should be in the locale files now.

I did a bit of testing with `Plural` but it felt a little weird to use it to do conditional text rendering. It worked for `en`, but I'm not sure if it was in the right form in the locale files. Would need a bit more testing to see if that would work.